src/config/locales/en/error.js

```diff
@@ -777,6 +777,10 @@ export const error = {
     }
   },
   currency: {
+    max: {
+      title: 'Currency value is too large',
+      message: 'Currency amounts over $2,147,483,647 are not supported.'
```

Comment:
I guess if you've had an epic bankruptcy, you're just now allowed to have a clearance.  😜